src/irr/builtin/CMakeLists.txt

```diff
@@ -2,22 +2,22 @@ function(irr_add_builtin_resource)
 
   # generate header file
   add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/build/builtinResources.h
-    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/buitlinHeaderGen.py ${CMAKE_CURRENT_BINARY_DIR}/build/builtinResources.h ${CMAKE_SOURCE_DIR} ${BUILTIN_RESOURCE_LIST}
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/builtinResources.h
+    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/buitlinHeaderGen.py ${CMAKE_CURRENT_BINARY_DIR}/builtinResources.h ${CMAKE_SOURCE_DIR} ${BUILTIN_RESOURCE_LIST}
     COMMENT "Generated built-in resource headers"
     VERBATIM
   )
 
   # generate cpp file with binary data of resources inside
   add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/build/builtinResourceData.cpp
-    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/builtinDataGen.py ${CMAKE_CURRENT_BINARY_DIR}/build/builtinResourceData.cpp ${CMAKE_SOURCE_DIR} ${BUILTIN_RESOURCE_LIST}
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/builtinResourceData.cpp
+    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/builtinDataGen.py ${CMAKE_CURRENT_BINARY_DIR}/builtinResourceData.cpp ${CMAKE_SOURCE_DIR} ${BUILTIN_RESOURCE_LIST}
     COMMENT "Generated built-in resource binary data"
     VERBATIM
   )
 
   # set up dependency
-  add_custom_target(builtin_resources DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/build/builtinResourceData.cpp ${CMAKE_CURRENT_BINARY_DIR}/build/builtinResources.h)
+  add_custom_target(builtin_resources DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/builtinResourceData.cpp" "${CMAKE_CURRENT_BINARY_DIR}/builtinResources.h")
```

Comment:
check if you have the paths right, you can use cmake `message` to check what `${CMAKE_CURRENT_BINARY_DIR}/builtin` evalutates to